### DISCUSSION TITLE
Load alarm diferent from android and iOS

### DISCUSSION THREAD

**Discussion by sandamil** [gdelataillade/alarm#33] [General]
If i load in iOS regresa alarms in screem same code in android i have to do hot reload to get alarms

**Comment by gdelataillade** [comment]
Hi @sandamil 

Sorry, I'm not sure to understand your issue. Can you provide more details please ? (steps to reproduce, expected behavior, plugin version, etc...)

**Reply by sandamil** [reply]
Sorry, I can't repeat the error because I solved it by forcing the screen refresh where I get the result and I haven't saved the moment it happened to me, basically it was with the same code for ios and android when returning to a screen from where I sent the date and load the alarms for that day, in ios it returned the alarms and in android I had to do a hot reload.
